platform/ui-next/src/contextProviders/ManagedDialog.tsx

```diff
@@ -60,6 +92,7 @@ const ManagedDialog: React.FC<ManagedDialogProps> = ({
                 transform: 'translate(0, 0)',
                 margin: 0,
                 animation: 'none',
+                transition: 'none',
```

Comment:
Ok @sedghi, I figured it out. We need `transition: none` too. Every once in a while even a blind squirrel finds a nut. 🤣